tests/e2e/workloads/amq/test_rgw_kafka_notifications.py

```diff
@@ -147,4 +151,25 @@ def test_rgw_kafka_notifications(self, bucket_factory):
                 "RGW bucket notification is not working as expected."
             )
 
+        # Validate the timestamp events
+        cmd = (
+            f"bin/kafka-console-consumer.sh --bootstrap-server {constants.KAFKA_ENDPOINT} "
+            f"--topic {self.kafka_topic.name} --from-beginning --timeout-ms 20000"
+        )
+        pod_list = get_pod_name_by_pattern(
+            pattern="my-cluster-zookeeper", namespace=constants.AMQ_NAMESPACE
+        )
+        zookeeper_obj = get_pod_obj(name=pod_list[0], namespace=constants.AMQ_NAMESPACE)
+        event_obj = zookeeper_obj.exec_cmd_on_pod(command=cmd)
+        log.info(f"Event obj: {event_obj}")
+        event_time = event_obj.get("Records")[0].get("eventTime")
+        format_string = "%Y-%m-%dT%H:%M:%S.%fZ"
+        try:
+            datetime.strptime(event_time, format_string)
+        except ValueError as ef:
+            log.error(
+                f"Timestamp event {event_time} doesnt match the pattern {format_string}"
+            )
+            raise ef
+
```

Comment:
Please add a log info message if the timestamp event matches the expected pattern